wordpress-mobile/WordPress-iOS
# Update WPKit to get Comment changes

Ref: #16685 
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/413

This updates WPKit to get changes to Comment parsing.

To test:

You can view/modify user roles at wordpress.com/people/team/yoursite.wordpress.com.

---
On a site where you cannot moderate comments (your role is Author or Contributor):
- Go to Site > Comments.
- Verify you can now see all Comments.
- Because the `can_moderate` flag is not used yet to disable moderation actions, they are still available. However, attempting to moderate a comment (approve, spam, trash) will yield an error message.

<kbd>![error](https://user-images.githubusercontent.com/1816888/123707570-23bf5d00-d827-11eb-8d52-b2a181a762f8.png)</kbd>

Note: Trashing a comment results in this. This will be resolved when the Trash action is disabled.

<img width="400" alt="trash" src="https://user-images.githubusercontent.com/1816888/123707479-01c5da80-d827-11eb-8957-8e83e0b3d4d9.png">

---
On a site where you can moderate comments (your role is Editor or Administrator):
- Go to Site > Comments.
- Verify you can see Comments.
- Verify you can moderate comments without error.

---
With any role:
- Go to any Comment details.
- Select `Edit`.
- Verify the content appears as expected, and does not contain HTML tags.

---
## Regression Notes
1. Potential unintended areas of impact
N/A. Feature incomplete.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature incomplete.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature incomplete.

PR submission checklist:
